localstorage/localstorage-todo.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Todo</title>
+    </head>
+
+    <body>
+
+        <form id="addPlaces">
+            <input type="text" placeholder="Add Place" />
+            <input type="submit" />
+        </form>
+        <ul id="listWrapper">
+        </ul>
+
+        <script>
+            let addPlaces = document.forms["addPlaces"];
+            let list = document.getElementById("listWrapper");
+
+            let placeArray = localStorage.getItem('place') ? localStorage.getItem('place').split(",") : [];
+
+            const createTodo = (inpVal) => {
+                let li = document.createElement("li");
+                let span = document.createElement("span");
+                let button = document.createElement("button");
+
+                li.appendChild(span);
+                li.appendChild(button);
+                list.appendChild(li);
+
+                span.textContent = inpVal;
+                button.textContent = "delete";
+            }
+
+            if (localStorage.getItem('place')) {
+                localStorage.getItem('place').split(",").map((item) => {
+                    return createTodo(item);
+                });
+            }
+
+            else {
+                addPlaces.addEventListener("submit", (e) => {
+                    e.preventDefault();
+                    let inpVal = addPlaces.querySelector("input[type='text']").value;
```

Comment:
Add an id to input so that you don't need to query the DOM everytime you need to reference the element again.